tests/publiccloud/download_repos.pm

```diff
@@ -88,9 +90,10 @@ sub run {
         # Failsafe 2: Ensure the repos are not empty (i.e. size >= 1 MB)
         $size = script_output('du -s ~/repos | awk \'{print$1}\'');
         die "Empty test repositories" if ($check_empty_repos && $size < 1000);
+
+        # The maintenance *.repo files all point to download.suse.de, but we are using dist.suse.de, so we need to rename the directory
+        assert_script_run("if [ -d ~/repos/dist.suse.de ]; then mv ~/repos/dist.suse.de ~/repos/download.suse.de; fi");
     }
-    # The maintenance *.repo files all point to download.suse.de, but we are using dist.suse.de, so we need to rename the directory
-    assert_script_run("if [ -d ~/repos/dist.suse.de ]; then mv ~/repos/dist.suse.de ~/repos/download.suse.de; fi");
     assert_script_run("cd");
```

Comment:
You can move this `cd` to the `else` condition one line above too.

Reply:
That was kept on purpose there, so that after the test run the VM will be in `/root`, regardless if updates have been downloaded or not.

Reply:
ok 😄 I've no hard feeling about it.